docs/src/using.md

```diff
@@ -65,17 +66,19 @@ The `CategoricalArray` still considers `"Old"` as a possible level even if it is
 
 To get rid of the `"Old"` group, just call the `droplevels!` function:
 
-```julia
+```jldoctest using
 julia> levels(x)
 3-element Array{String,1}:
  "Young" 
  "Middle"
  "Old"   
 
 julia> droplevels!(x)
-2-element Array{String,1}:
+4-element CategoricalArrays.CategoricalArray{String,1,UInt32,String,Union{}}:
```

Comment:
Yes, I've changed that for consistency, since generally mutating functions return their input. You can easily wrap the call inside `levels(...)` if needed, but the contrary wasn't possible.